.github/workflows/build.yaml

```
@@ -54,3 +54,61 @@ jobs:
         run: "make generate \nif ! git diff --exit-code --quiet ; then \n  echo\n  echo\n  git diff --stat \"HEAD\"\n  echo\n  echo\n  echo 'ERROR: Lint tools caused changes to the working dir. '\n  exit 1\nfi\n"
       - name: make test
         run: make test
+  e2e:
+    needs: build
+    if: "${{ github.event.action != 'labeled' || github.event.label.name == 'tests: run' }}"
+    name: Run E2E tests on Google Cloud
```

nit: shorten name for cleaner checks in UI
```suggestion
    name: e2e tests
```

---

Done.